be/src/exec/pipeline/pipeline_driver_poller.cpp

```diff
@@ -159,13 +159,52 @@ void PipelineDriverPoller::run_internal() {
 
 void PipelineDriverPoller::add_blocked_driver(const DriverRawPtr driver) {
     std::unique_lock<std::mutex> lock(_global_mutex);
+#if BE_TEST
+    VLOG_ROW << "Add to blocked driver:" << driver->to_readable_string();
+#endif
     _blocked_drivers.push_back(driver);
     driver->_pending_timer_sw->reset();
     _cond.notify_one();
 }
 
+void PipelineDriverPoller::add_parked_driver(const DriverRawPtr driver) {
+    std::unique_lock<std::mutex> lock(_global_mutex);
+#if BE_TEST
+    VLOG_ROW << "Add to parked driver:" << driver->to_readable_string();
+#endif
+    _parked_drivers.push_back(driver);
+    driver->_pending_timer_sw->reset();
+    _cond.notify_one();
```

Comment:
Should use `_global_parked_mutex` instead of `_global_mutex`, and not use `_cond.notify_one` because nothing is put to `blocked_drivers`?

Reply:
fixed.